Source/WebKit/Shared/ShareableBitmap.cpp

```diff
@@ -118,7 +118,7 @@ RefPtr<ShareableBitmap> ShareableBitmap::createFromImageDraw(NativeImage& image)
 
 RefPtr<ShareableBitmap> ShareableBitmap::create(const ShareableBitmapHandle& handle, SharedMemory::Protection protection)
 {
-    auto sharedMemory = SharedMemory::map(handle.m_handle, protection);
+    auto sharedMemory = SharedMemory::map(WTFMove(handle.m_handle), protection);
```

Comment:
Unfortunately this indicates a contagiousness of the change:
All the types that contain SharedMemoryHandle must now support move semantics also.
So before on Cocoa, SharedMemory::map did not mutate the handle. This meant that the ShareableBitmapHandle handle here would stay intact and operational after ShareableBitmap::create(). Now, after your change, the m_handle is moved away and the ShareableBitmapHandle is rendered inoperable.
It is unknown if any code relied on the ShareableBitmapHandle staying operable.

Thus to prove correctness, the approach would be to first make ShareableBitmapHandle movable, land that, and then continue with this patch.

`RefPtr<ShareableBitmap> ShareableBitmap::create(const ShareableBitmapHandle& handle, SharedMemory::Protection protection)`
->
`RefPtr<ShareableBitmap> ShareableBitmap::create(ShareableBitmapHandle&& handle, SharedMemory::Protection protection)`